lib/convert.ts

```diff
@@ -191,7 +190,7 @@ export default class Convert {
       case 'Info':
         return ls.DiagnosticSeverity.Information;
       default:
-        (type: empty);
+        // (type: empty);
```

Comment:
This line gave an error in the TypeScript compiler, not sure what purpose it serves.

Reply:
Assuming that the purpose was to assert that the switch was exhaustive over DiagnosticType, one way of writing this in typescript is:

```typescript
const never: never = type;
throw Error(`Unexpected diagnostic type ${never}`);
```

The `never` type is a type with no legal values, so that assignment will fail if type inference thinks that there are any legal values for `type` at that point in the program.

(Using `never` instead of `types` in the error ensures that it doesn't look like `never` is an unused variable when linting or compiling)